src/util.js

```diff
@@ -16,6 +16,31 @@ function loadQuery(query) {
   return fs.readFileSync(path.resolve(__dirname, 'queries', `${query.replace(/^\//, '')}.sql`)).toString();
 }
 
+function cleanQueryParams(query, params) {
```

Comment:
A minimum of JS doc would be useful. I quickly read the code and the tests but I am still not 100% sure what this function is really doing.

Reply:
Got it! I made another Issue for this!

Reply:
https://github.com/adobe/helix-run-query/issues/44